Sources/MapboxNavigation/Resources/ar.lproj/Localizable.strings

```diff
@@ -1,6 +1,276 @@
+
+/* Title on arrival action sheet */
+"CARPLAY_ARRIVED" = "You have arrived";
+
+/* Message on arrival action sheet */
+"CARPLAY_ARRIVED_MESSAGE" = "What would you like to do?";
+
+/* Title on continue button in CarPlay */
+"CARPLAY_CONTINUE" = "Continue";
+
+/* Name of the waypoint associated with the current location */
+"CARPLAY_CURRENT_LOCATION" = "Current Location";
+
+/* Title for dismiss button */
+"CARPLAY_DISMISS" = "Dismiss";
+
+/* Title for end navigation button */
+"CARPLAY_END" = "End";
+
+/* Title on the exit button in the arrival form */
+"CARPLAY_EXIT_NAVIGATION" = "Exit navigation";
+
+/* Title for feedback template in CarPlay */
+"CARPLAY_FEEDBACK" = "Feedback";
+
+/* Title for start button in CPTripPreviewTextConfiguration */
+"CARPLAY_GO" = "Go";
+
+/* Title for alternative routes in CPTripPreviewTextConfiguration */
+"CARPLAY_MORE_ROUTES" = "More Routes";
+
+/* Title for mute button */
+"CARPLAY_MUTE" = "Mute";
+
+/* Title for overview button in CPTripPreviewTextConfiguration */
+"CARPLAY_OVERVIEW" = "Overview";
+
+/* Title for trip preview back button */
+"CARPLAY_PREVIEW_BACK" = "Back";
+
+/* Title for rating template in CarPlay */
+"CARPLAY_RATE_RIDE" = "Rate your ride";
+
+/* Title on rate button in CarPlay */
+"CARPLAY_RATE_TRIP" = "Rate your trip";
+
+/* Alert title that shows when feedback has been submitted */
+"CARPLAY_SUBMITTED_FEEDBACK" = "Submitted";
+
+/* Title for unmute button */
+"CARPLAY_UNMUTE" = "Unmute";
+
+/* Specific route feedback that audio guidance provided was confusing. */
+"CONFUSING_AUDIO_FEEDBACK" = "Confusing audio";
+
+/* Specific route feedback that audio guidance was provided too early before a maneuever. */
+"CONFUSING_AUDIO_GUIDANCE_TOO_EARLY_FEEDBACK" = "Guidance too early";
+
+/* Specific route feedback that audio guidance was provided too late for a maneuever. */
+"CONFUSING_AUDIO_GUIDANCE_TOO_LATE_FEEDBACK" = "Guidance too late";
+
+/* Specific route feedback that audio guidance instruction is missing */
+"CONFUSING_AUDIO_INSTRUCTION_MISSING_FEEDBACK" = "Instruction missing";
+
+/* Specific route feedback that audio guidance used incorrect pronunciation. */
+"CONFUSING_AUDIO_PRONUNCIATION_INCORRECT_FEEDBACK" = "Pronunciation incorrect";
+
+/* Specific route feedback that audio guidance repeated a road name. */
+"CONFUSING_AUDIO_ROADNAME_REPEATED_FEEDBACK" = "Road name repeated";
+
+/* Dismiss button title on the steps view */
+"DISMISS_STEPS_TITLE" = "إغلاق";
+
+/* Label indicating precise location is off and needs to be turned on to navigate */
+"ENABLE_PRECISE_LOCATION" = "Enable precise location to navigate";
+
+/* Title used for arrival */
+"END_OF_ROUTE_ARRIVED" = "لقد وصلت";
+
+/* Comment Placeholder Text */
+"END_OF_ROUTE_TITLE" = "كيف يمكننا التحسين؟";
+
+/* Number string with an explicit '+' sign. */
+"EXPLICITLY_POSITIVE_NUMBER" = "+%@";
+
+/* Alternatives selection note about a faster route time interval in any unit. */
+"FASTER_ALTERNATIVE" = "%@ faster";
```

Comment:
Can we remove the labels in English?  English would be used by default for those not presented